docs/rfcs/0001-openedx-auth-integration.md

```diff
@@ -0,0 +1,26 @@
+## Open edX Auth Integration
+
+#### Abstract
+
+We need to be able to authenticate MIT xPro users with our hosted version of Open edX.
+
+- User should not have to take any manual effort (e.g. entering a password) to login to Open edX
+- This will _probably_ need to be some flavor of OAuth where MIT xPro is the OAuth Service Provider
+
+#### Architecture Changes
+
+##### Write a OAuth2 Provider library for Open edX
+
+Open edX supports additional custom OAuth2 providers that `python-social-auth` doesn't support out of the box as documented [here](https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/tpa/tpa_integrate_open/tpa_oauth.html#additional-oauth2-providers-advanced). This would involve writing and publishing to pypi a module that has our implementation in it.
+
+##### Enable MIT xPro as an OAuth2 Service Provider
+
+The most straightforward solution is to use [`django-oauth-toolkit`](https://django-oauth-toolkit.readthedocs.io/en/latest/) to accomplish this.
```

Comment:
I see that edx-platform currently uses both django-oauth2-provider and django-oauth-toolkit. What's your reason for choosing django-auth-toolkit?

Reply:
`django-auth-toolkit` is actively maintained, `django-oauth2-provider` doesn't appear to have a commit in the last 5 years.